plugins/inputs/openldap/openldap_test.go

```diff
@@ -140,20 +308,40 @@ func commonTests(t *testing.T, o *Openldap, acc *testutil.Accumulator) {
 }
 
 func TestOpenldapReverseMetricsIntegration(t *testing.T) {
-	t.Skip("skipping test as unable to read LDAP response packet: unexpected EOF")
+	if testing.Short() {
+		t.Skip("Skipping integration test in short mode")
+	}
+
+	container := testutil.Container{
+		Image:        "bitnami/openldap",
+		ExposedPorts: []string{"1389"},
+		Env: map[string]string{
+			"LDAP_ADMIN_USERNAME": "manager",
+			"LDAP_ADMIN_PASSWORD": "secret",
+		},
+		WaitingFor: wait.ForListeningPort("1389/tcp"),
+	}
+	err := container.Start()
+	require.NoError(t, err, "failed to start container")
+	defer func() {
+		require.NoError(t, container.Terminate(), "terminating container failed")
+	}()
+
+	port, err := strconv.Atoi(container.Port)
+	require.NoError(t, err)
 
 	o := &Openldap{
-		Host:               testutil.GetLocalHost(),
-		Port:               389,
+		Host:               container.Address,
+		Port:               port,
 		SSL:                "",
 		InsecureSkipVerify: true,
-		BindDn:             "cn=manager,cn=config",
+		BindDn:             "CN=manager,DC=example,DC=org",
 		BindPassword:       "secret",
 		ReverseMetricNames: true,
 	}
 
 	var acc testutil.Accumulator
-	err := o.Gather(&acc)
+	err = o.Gather(&acc)
 	require.NoError(t, err)
-	require.True(t, acc.HasInt64Field("openldap", "connections_total"), "Has an integer field called connections_total")
+	require.True(t, acc.HasField("openldap", "connections_total"), "Has an integer field called connections_total")
```

Comment:
Any reason to relax the test on that field?

Reply:
oops! good catch - I don't remember why I did that

Reply:
fix pushed